foundation/foundation-archetype/test-project-archetype.sh

```diff
@@ -308,13 +312,12 @@ cd ../..
 echo -e "\nINFO: Generating project structure for ml pipelines\n"
 runBuildAndUpdateDeploy
 
-echo -e "\nINFO: updating the test-generator-deploy/pom.xml based on static code as it's hard to predict\n"
-plugins=$(esc < ../../../plugins-to-add-to-archetype_test-generator-deploy_pom.xml)
-sub "s/ *<\/plugins>/$plugins        <\/plugins>/" test-generator-deploy/pom.xml
+echo -e "\nINFO: Running final build to ensure success"
+./mvnw clean install
```

Comment:
C: Exit with non-zero code if build fails
```suggestion
./mvnw clean install || { echo -e '\n\n\t**** MAVEN BUILD FAILED ****\n\n' ; exit 1; }
```